pkg/gce-pd-csi-driver/utils_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
I merged these two lines in https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/827, could you rebase from master please?

Reply:
Oh, I see, gofmt got confused. Thanks.